core/migrations/0018_floorsection.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.0.1 on 2018-07-29 22:03
```

Comment:
After the changes to the model, ensure you check in only one migration file reflecting the changes.